### ISSUE TITLE
Remove "connected peers count" message

### ISSUE THREAD

**Issue by nkuba** [keep-network/keep-core#3377]
As the client exposes metrics now we could remove logging the number of connected peers:
https://github.com/keep-network/keep-core/blob/6aa2f289ac67d7bd4f06150b3e3a62f77c0df2c2/pkg/net/libp2p/libp2p.go#L241-L242

Thoughts?

**Comment by pdyraga** [comment]
That makes sense to me.

**Comment by michalinacienciala** [comment]
Yes, makes sense to me as well. Peer data can be easily checked in metrics, and older data can be seen via Grafana dashboard. Having peer data in logs unnecessarily clutters them...

**Comment by lionakhnazarov** [comment]
Closing as stale. Targets the legacy keep-core v1 (random beacon / KEEP staking) architecture, superseded by the Threshold v2 stack. Not actionable against current main.
